app/views/teacher_training_adviser/steps/completed.html.erb

```diff
@@ -21,28 +21,36 @@
           If we are unable to contact you at this time, we will try to contact you a further 2 times. If we are still unable to contact you, you will have to sign up again.
         </p>
         <% else %>
-        <p>
-          You’ll get an email from us soon which will outline the next steps for you to talk to an adviser.
-        </p>
-        <p>
-          You must respond to the email within 6 weeks of receiving it otherwise you will have to sign up again.
-        </p>
+
+        <p>An adviser will email you within 3 working days.</p>
+
+        <p>You need to reply to the email to be assigned an adviser.</p>
```

Comment:
Is this agreed with Sarah too @Tomcowan1?

Reply:
Yes she did :-)